src/infiniteGrid.js

```diff
@@ -59,6 +59,7 @@ eg.module("infiniteGrid", ["jQuery", eg, window, document], function($, ns, glob
 		"append": "append",
 		"prepend": "prepend"
 	};
+	var RETRY = 3;
```

Comment:
Retrying three times is stopgap solution. 
Okay for now, but need to find some more smart solution later.